### DISCUSSION TITLE
Show MMR Delta and The match that the player ranked up

### DISCUSSION THREAD

**Discussion by elbegnyamdorj** [ValveSoftware/Dota2-Gameplay#9967] [Ideas]
Its a pretty simple yet user friendly feature request. After the new confidence rank system I realized the gain and loss of mmr is not static like old time(30 solo, 20 party). So showing it in the match detail or list would be nice. 
The second one is similar to Valorant, In the match list, show at which match the player has ranked up. Just a green little upnotch ^ or red down notch implying to rank down. 
Thanks.

**Comment by EmilyTrantow** [comment]
I completely agree! Displaying the MMR gain/loss in match details would enhance user experience significantly. It’s crucial for players to understand their progress, especially with the new confidence rank system. Speaking of progress, if you're looking for a fun way to improve your skills, check out [Snow Rider](https://snowridergame.io) ! It’s a great game that sharpens your reflexes while being entertaining.

**Reply by katchwi-arch** [reply]
bet spammers at it again... u are ruinning alreaaady the game with dota plus matchmaking, isnt it enough? u want to be completly sure u win so u can place ur bets, right?

**Comment by Yunkre** [comment]
They evoke feelings of fear, excitement, and the need to make quick decisions like enormous mechanical obstacle courses.
[moto x3m](https://motox3mgame.org/)
